library/macos/FLEMethodChannel.h

```diff
@@ -69,14 +69,14 @@ extern NSString const *_Nonnull FLEMethodNotImplemented;
  *
  * The arguments, if any, must be encodable using this channel's codec.
  */
-- (void)invokeMethod:(nonnull NSString *)method arguments:(id _Nullable)arguments;
```

Comment:
Should the qualifiers in FLEBasicMessageChannel.h change as well?

Reply:
The nullability annotations on FLEBasicMessageChannel are using this style. The typedefs (as with those at the top of this file) can't, because this form isn't usable in all language constructions.